src/course-user/course-user.service.ts

```diff
@@ -1,4 +1,37 @@
 import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { CourseUserEntity } from '../entities/course-user.entity';
+import { CourseUserRepository } from './course-user.repository';
+import { CourseUserDto } from './dto/course-user.dto';
 
 @Injectable()
-export class CourseUserService {}
+export class CourseUserService {
+  constructor(
+    @InjectRepository(CourseUserRepository) private courseUserRepository: CourseUserRepository,
+  ) {}
+
+  async getCourseUser({ courseId, userId }: CourseUserDto): Promise<CourseUserEntity[]> {
```

Comment:
why does this return a list of courseUsers? it should only ever return 1 right? 
then in `session-user.service.ts` we can use `courseUser` instead of `courseUser[0]`